packages/zoe/test/unitTests/contracts/test-multipoolAutoswap.js

```diff
@@ -388,15 +388,15 @@ test('multipoolAutoSwap with valid offers', async t => {
   );
   t.deepEqual(
     await simoleanR.issuer.getAmountOf(bobSimsPayout3),
-    simoleans(0),
-    `bob gets no simoleans back`,
+    simoleans(9),
+    `bob gets 9 simoleans back because 74 was more than required`,
```

Comment:
Giving money back if more was provided than is necessary is a nice touch, but I'm not sure this is the right thing to do here. It looks like we are taking the `wanted` amountOut as the "true" statement of what the user wants. But if that were the case, why wouldn't they use SwapOut? I think in SwapIn, the user is saying, I want to give all of these tokens and get the most I can back out, at least the `wanted` amountOut.